src/main/java/dev/meyi/bn/json/resp/BazaarItem.java

```diff
@@ -31,6 +31,10 @@ public BazaarSubItem(int amount, double pricePerUnit, int orders) {
       this.pricePerUnit = pricePerUnit;
       this.orders = orders;
     }
+
+    public double getPriceWithTax(){
+      return pricePerUnit * 0.9875d;
+    }
```

Comment:
This is good, but we might also want to consider checking the api for the bazaar tax profile upgrade. This can be done at the same time as the collections check and should be set in the configuration. 

For now though, it's fine. just something we might want to do in the future to keep the tax accurate.